src/web/app/pages-instructor/instructor-session-individual-extension-page/instructor-session-individual-extension-page.component.ts

```diff
@@ -152,16 +148,14 @@ export class InstructorSessionIndividualExtensionPageComponent implements OnInit
   private getAllStudentsOfCourse(): void {
     this.studentService
       .getStudentsFromCourse({ courseId: this.courseId })
-      .pipe(
-        finalize(() => {
-          this.isLoadingAllStudents = false;
-        }),
+      .pipe(finalize(() => { this.isLoadingAllStudents = false; }),
         map((students: Students) => {
-          this.studentsOfCourse = students.students.map((student) => this.mapStudentToStudentModel(student));
+          return students.students.map((student) => this.mapStudentToStudentModel(student));
         }),
```

Comment:
Can do without the return here, just `(students: Students) => students.students.map...`

If you want to get rid of the awkward `students.students` you can even add another map.

```
map((students:Students) => students.students),
map((students:Student[]) => students.mapStudentToStudentModel),
```

Reply:
Thanks, I just did `({ students }: Students) => students.map`

Reply:
Actually, I refactored it. So the `mapStudentToStudentModel` is now `mapStudentsToStudentModels`. It's cleaner to look at now